autoread.js

```diff
@@ -113,4 +115,21 @@ function handleCheckbox(currentLine){
 
 function handleSublist(currentLine){
 	return Array.of(...currentLine.childNodes).filter(x=>x.nodeName != "OL" && x.nodeName != "UL").map(x=>x.textContent).join('');
+}
+
+function orientationCheck(){
+		let sBar = document.getElementById("sidebar");
+		let sBox = document.getElementById("speechbox");
+		
+		if(window.innerHeight + 100 > window.innerWidth){
```

Comment:
I love the condition of "portrait vs landscape". I think that'll work nicely.

Reply:
what about ipad? do we want that to be considered "portrait" and have this applied?

Reply:
Maybe a manual override button or setting?